lambdas/src/apis/profiles/controllers/profiles.ts

```diff
@@ -88,6 +88,36 @@ export async function getIndividualProfileById(
   sendProfilesResponse(res, profiles, profilesCacheTTL, true)
 }
 
+export async function getProfilesByIdPost(
+  theGraphClient: TheGraphClient,
+  contentClient: SmartContentClient,
+  ensOwnership: EnsOwnership,
+  wearables: WearablesOwnership,
+  profilesCacheTTL: number,
+  req: Request,
+  res: Response
+): Promise<Response<any, Record<string, any>> | undefined> {
+  // Method: POST
+  // Path: /lambdas/profiles
+  // Body: { ids: string[] }
+
+  const profileIds: string[] = req.body.ids ?? []
```

Comment:
This would break the code if `ids` param is not passed in
```suggestion
  const profileIds: string[] = req.body?.ids ?? []
```

Reply:
fixed